src/i18n/locales/da.json

```diff
@@ -1,10 +1,24 @@
 {
-    "welcome": "Velkommen!",
-    "sex": "Køn",
-    "lgbtq": "LGBT+",
+    "homepage": {
+        "title": "Velkommen til Centralen for Asylansøgere"
+    },
+
+    "nav": {
+        "Home": "Hjem",
```

Comment:
this looks perfect! only thing is super nitpick but I think the convention here is to use camelCase. so maybe just change the keys to camel case and then it should be good to merge :)